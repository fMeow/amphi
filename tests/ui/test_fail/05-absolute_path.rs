#![allow(dead_code)]
use amphi::amphi;

#[amphi(async_only, path="/home/user/project/lib.rs")]
mod amphi{
    pub async fn my_fn() -> bool {
        true
    }
}

fn main() {
}
