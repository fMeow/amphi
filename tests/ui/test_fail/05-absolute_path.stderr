error: Absolute path is not allowed. Please use relative path.
 --> $DIR/05-absolute_path.rs:4:26
  |
4 | #[amphi(async_only, path="/home/user/project/lib.rs")]
  |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
